### ISSUE TITLE
Error message may be localized

### ISSUE THREAD

**Issue by eserte** [sbakker/perl-Term-CLI#4]
t/075-Term-CLI-History.t may fail if a non-English locale is in effect. For example with de_DE.*:
```
#   Failed test 'error on failed read_history is set correctly'
#   at t/075-Term-CLI-History.t line 51.
#   (in Term_CLI_History_test->check_history)
#                   '/does/not/exist: Datei oder Verzeichnis nicht gefunden'
#     doesn't match '(?^ui:no such file or directory)'
# Looks like you failed 1 test of 12.
t/075-Term-CLI-History.t ................ 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/12 subtests 
```

**Comment by eserte** [comment]
The localization happen only with perl <= 5.20, it seems.

**Comment by sbakker** [comment]
I though I removed all these English-centric checks. I noticed form the CPAN testers output that this one was still there. The 0.04002 release should have fixed that one as well.
As for the Perl version, that's interesting. The error that is being checked comes from "$!", so perhaps something changed between 5.20 and onwards...

**Comment by sbakker** [comment]
Hah! Found it: http://search.cpan.org/dist/perl-5.22.0/pod/perldelta.pod#%22$!%22_text_is_now_in_English_outside_the_scope_of_use_locale

### "$!" text is now in English outside the scope of use locale

Previously, the text, unlike almost everything else, always came out based on the current underlying locale of the program. (Also affected on some systems is "$^E".) For programs that are unprepared to handle locale differences, this can cause garbage text to be displayed. It's better to display text that is translatable via some tool than garbage text which is much harder to figure out.

**Comment by sbakker** [comment]
Hah! Found it: http://search.cpan.org/dist/perl-5.22.0/pod/perldelta.pod#%22$!%22_text_is_now_in_English_outside_the_scope_of_use_locale

### "$!" text is now in English outside the scope of use locale

Previously, the text, unlike almost everything else, always came out based on the current underlying locale of the program. (Also affected on some systems is "$^E".) For programs that are unprepared to handle locale differences, this can cause garbage text to be displayed. It's better to display text that is translatable via some tool than garbage text which is much harder to figure out.

**Comment by sbakker** [comment]
Fixed in 0.04002.
